manifests/client.pp

```diff
@@ -15,14 +15,26 @@
   $verb           = 3,
   $cipher         = 'AES-192-CBC',
   $compression    = 'lzo',
-  $openvpn_group  = $openvpn::params::openvpn_group,
-  $openvpn_user   = $openvpn::params::openvpn_user,
+  $openvpn_group  = '',
+  $openvpn_user   = '',
   $tls_auth_key   = undef,
   $custom_options = [],
 ) {
 
   include openvpn
 
+  if $openvpn_group != '' {
```

Comment:
I'm not clear why these two blocks are necessary.  If we have received a value for openvpn_group and/or openvpn_user, we'll use those, but the default already references the params value.